cmd/options.go

```diff
@@ -26,9 +27,13 @@ func (o *Options) MaxScrapeSizeBytes() (int64, error) {
 
 func (o *Options) AddFlags(app extkingpin.AppClause) {
 	app.Flag("scrape-url", "URL to scrape metrics from").
-		Required().
+		Default("").
 		StringVar(&o.ScrapeURL)
 
+	app.Flag("scrape-file", "File to scrape metrics from").
```

Comment:
You said in the PR description, and sounds like a better idea. We should namespace these args with `.` instead of `-`, so use `scrape.url` and `scrape.file`, as it is common in other tools like Thanos.